caravel/assets/javascripts/SqlLab/components/TabbedSqlEditors.jsx

```diff
@@ -131,11 +136,18 @@ class TabbedSqlEditors extends React.Component {
         >
           <div className="panel panel-default">
             <div className="panel-body">
-              <SqlEditor
-                queryEditor={qe}
-                latestQuery={latestQuery}
-                database={database}
-              />
+              {isSelected ?
```

Comment:
this can be written more simply, we don't need the `condition ? then : else`ternary here.

```
{isSelected &&
  <SqlEditor 
    ...
  />
}
```